modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/mesa_mobs.dm

```diff
@@ -549,6 +549,8 @@
 	desc = "A 5.56mm surplus bullet casing."
 	projectile_type = /obj/projectile/bullet/a556/weak
 
+#define MOB_PLACER_RANGE CONFIG_GET(string/default_view) + 1 // One more tile than the biggest viewrange we have.
```

Comment:
Actually I think that config might not give us the thing we actually want, as that seems to be a string that says `15x15`.

I'd probably say something like 8 would be a good value here, and if other servers have for some reason an even bigger distance, well... Sucks to be them I guess.

Unless we can find the define that's already saying the size of our widescreen.

Reply:
The default *should be* 21x15, and the non default should be 15x15, as a fresh server defaults to 21x15.